beta/examples/tensorflow/classification/main.py

```diff
@@ -11,19 +11,25 @@
  limitations under the License.
```

Comment:
Please, add achieved results and config files to reproduce results for TF examples.

Reply:
done